.github/CODEOWNERS

```
@@ -164,6 +164,7 @@ packages/react-components/react-toolbar @microsoft/teams-prg @chpalac @ling1726
 packages/react-components/react-portal-compat @microsoft/teams-prg
 packages/react-components/react-alert @microsoft/teams-prg
 packages/react-components/react-portal-compat-context @microsoft/teams-prg
+#packages/react-components/react-theme-sass @microsoft/teams-prg
```

This should be uncommented, right?

```suggestion
packages/react-components/react-theme-sass @microsoft/teams-prg
```